.github/workflows/ci.yml

```
@@ -81,85 +81,88 @@ jobs:
       matrix:
         include:
         ### Analysis builds
+        - name: linux_gcc-10_debug_analysis-omp5
```

Could you please use an underscore here and in the renamed clang job?
```suggestion
        - name: linux_gcc-10_debug_analysis_omp5
```

---

Done,.